src/sec_edgar_bulker/core/downloader.py

```
@@ -0,0 +1,500 @@
+            await self.session.close()
+    
+    async def __aenter__(self):
+        """Enter async context."""
+        self.session = aiohttp.ClientSession()
+        return self
+    
+    async def __aexit__(self, exc_type, exc_val, exc_tb):
+        """Exit async context."""
+        await self.cleanup()
+    
+    def __del__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__del__`

Add return type annotation: `None`

(ANN204)